source/unwind.js

```diff
@@ -22,7 +22,7 @@ import _assoc from './internal/_assoc.js';
  *   hobbies: ['Golf', 'Hacking'],
  *   colors: ['red', 'green'],
  * });
- * //=> [
+ * // [
  * //     { name: 'alice', hobbies: 'Golf', colors: ['red', 'green'] },
```

Comment:
Now I feel a bit annoying, 😆 -- the convention is 2 spaces.

Reference: https://ramdajs.com/docs/#groupBy